src/rust/ide/view/graph-editor/src/component/edge.rs

```diff
@@ -13,11 +14,12 @@ use ensogl::display::traits::*;
 use ensogl::display;
 use ensogl::gui::component::ShapeViewEvents;
 use ensogl::gui::component;
+use ensogl_shape_utils::dynamic_color::DynamicColor;
+use ensogl_shape_utils::dynamic_color;
 use ensogl_theme as theme;
 
-use super::node;
 use crate::component::node::NODE_HEIGHT;
-
+use super::node;
```

Comment:
Imports order + I think we should rather have `use crate::component::node` than `super::node` at top of the file.

Remember to put 3 empty lines after imports